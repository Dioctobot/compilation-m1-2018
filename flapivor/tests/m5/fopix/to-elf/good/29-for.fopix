def ref (v) =
  val b = allocate_block (1);
  b[0] := v;
  b

def read (r) =
  r[0]

def write (r, v) =
  r[0] := v

val main =
    val i = ref (0);
    while (read (i) <= 10) do
      val z = read (i) + 1;
      write (i, z);
      print_int (read (i))
    end