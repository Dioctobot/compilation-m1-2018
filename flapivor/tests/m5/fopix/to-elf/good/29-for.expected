Process exited with status exited(0).
STDOUT:
1234567891011
STDERR:



